tools/hive/kerberos/conf/krb5.conf

```diff
@@ -0,0 +1,10 @@
+[libdefaults]
+default_realm = LOCAL
+dns_lookup_realm = true
+dns_lookup_kdc = true
+udp_preference_limit = 1
+
+[realms]
+LOCAL = {
+  kdc = kdc.marathon.mesos:2500
```

Comment:
In our other applications, we use a different endpoint here Should we make this configurable too?

Reply:
Hm, this is just for testing. Not sure we need to make this configurable.

Reply:
My concern is that it is different to how we handle KDC in all our other applications. If we need to move forward with this though, I'm not going to block the PR, but we should consider creating a follow-up ticket to unify this.